src/frontend/hooks/useDokument.ts

```diff
@@ -36,6 +45,34 @@ const useDokument = () => {
         return bytes;
     };
 
+    const hentDistribusjonskanal = (personIdent: string) => {
+        nullstillDistribusjonskanal();
+        request<{ ident: string }, Distribusjonskanal>({
+            method: 'POST',
+            data: { ident: personIdent },
+            url: `/familie-ba-sak/api/dokument/distribusjonskanal`,
+        })
+            .then((response: Ressurs<Distribusjonskanal>) => {
+                if (response.status === RessursStatus.SUKSESS) {
+                    settDistribusjonskanal(byggDataRessurs(response.data));
+                } else if (
+                    response.status === RessursStatus.FEILET ||
+                    response.status === RessursStatus.FUNKSJONELL_FEIL ||
+                    response.status === RessursStatus.IKKE_TILGANG
+                ) {
+                    settDistribusjonskanal(response);
+                } else {
+                    settDistribusjonskanal(
+                        byggFeiletRessurs('Ukjent feil, kunne ikke hente distribusjonskanal.')
+                    );
+                }
```

Comment:
Med unntak av statusene `HENTER` og `IKKE_HENTET` er ikke dette det samme som
```suggestion
                settDistribusjonskanal(response.data);
```

Reply:
Og kan vi egentlig få HENTER og IKKE_HENTET fra backend?

Reply:
Reagerte litt på det selv da jeg kopierte denne koden, og ser veldig ut som det kan forenkles, ja